src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1133,7 +1133,22 @@ private uint getClassSize(CORINFO_CLASS_STRUCT_* cls)
         }
 
         private uint getClassAlignmentRequirement(CORINFO_CLASS_STRUCT_* cls, bool fDoubleAlignHint)
-        { throw new NotImplementedException("getClassAlignmentRequirement"); }
+        {
+            if (_compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.ARM ||
```

Comment:
This should not be special cased for ARM. The implementation should work everywhere.